pkgs/development/libraries/gstreamer/bad/default.nix

```diff
@@ -85,6 +137,38 @@ stdenv.mkDerivation rec {
     # TODO: mjpegtools uint64_t is not compatible with guint64 on Darwin
     ++ optional (!stdenv.isDarwin) mjpegtools;
 
+  mesonFlags = [
+    # Enables all features, so that we know when new dependencies are necessary.
+    "-Dauto_features=enabled"
+
+    "-Dexamples=disabled" # requires many dependencies and probably not useful for our users
+
+    "-Ddts=disabled" # required `libdca` library not packaged in nixpkgs as of writing, and marked as "BIG FAT WARNING: libdca is still in early development"
+    "-Dfaac=${if faacSupport then "enabled" else "disabled"}"
+    "-Dgsm=disabled" # as of writing, with `gsm` in `buildInputs` we get "GSM plugin is enabled: found libgsm but no headers"; gsm packaging problem?
+    "-Diqa=disabled" # required `dssim` library not packaging in nixpkgs as of writing
+    "-Dmsdk=disabled" # not packaged in nixpkgs as of writing / no Windows support
+    "-Dmusepack=disabled" # as of writing, with `libmpcdec` in `buildInputs` we get "Could not find libmpcdec header files, but Musepack was enabled via options"; libmpcdec packaging problem?
```

Comment:
We have the header in `libmpc/mpcdec.h` instead of `mpc/mpcdec.h` like [Arch does](https://www.archlinux.org/packages/extra/x86_64/libmpcdec/). Not sure who is at fault.

Reply:
The situation looks difficult. There are apparently 2 things called `libmpcdec` from the same author:

* http://svn.musepack.net/libmpcdec/trunk/src/
* http://svn.musepack.net/libmpc/trunk/include/mpc/

I'll postpone this for now and add a comment.